src/restore.c

```diff
@@ -66,18 +66,16 @@ static int read_restore (hashcat_ctx_t *hashcat_ctx)
   }
 
   // we only use these 2 checks to avoid "tainted string" warnings
-
-  if (rd->argc < 1)
+  // raise upper bound to 5000 when --force'd
```

Comment:
also add a comment here why exactly this 5000 number was choosen, otherwise we will risk to not remember where exactly this value comes from in the future

Reply:
(and do not forget to remove the "when --force'd" if you agree to get rid of the --force dependency)